lib/xcsoar/mapgen/server/worker.py

```diff
@@ -24,10 +25,10 @@ def __send_download_mail(self, job):
 To: {to}
 Subject: XCSoar Map Generator - Download ready ({name}.xcm)
 
-The XCSoar Map Generator has finished your map.
-It can be downloaded at {url}
+The XCSoar Map Generator has finished your map: {name}
+It can be downloaded at {protocol}://{domain}{url}
 This link is valid for 7 days.
-'''.format(to=job.description.mail, name=job.description.name, url=job.description.download_url)
+'''.format(to=job.description.mail, name=job.description.name, protocol=mapgen['protocol'],domain=mapgen['domain'],url=job.description.download_url)
```

Comment:
I would put spaces between the arguments here.

Reply:
But I guess I see your point in not doing so, keeping those related ones together. Not convinced it improves readability though :)

Reply:
Uniformity makes sense. Thx.